helm/mockserver/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: mockserver
-version: "5.5.1"
+version: "1.0"
```

Comment:
I believe this change is semantically more correct and is aligned to how you attended the Helm chart to be used. Specifically that you set the container version based on the "AppVersion" which is independent of the Helm chart "Version".

Reply:
The version of all artefacts is linked to reduce complexity this means that the version of the helm chart should be the same is the underlying version of MockServer.  This is important to reduce complexity both for those implementing MockServer and for users of MockServer so they can easily understand what artefact versions work with other artefact versions, i.e client, docker, helm, server, etc.

Reply:
Also this value drives the generated tar file name and path